apisix/plugins/jwt-auth.lua

```diff
@@ -200,6 +211,51 @@ function _M.rewrite(conf, ctx)
 end
 
 
+local function sign_jwt_with_HS(key, auth_conf)
+    local auth_secret = get_secret(auth_conf)
+    local jwt_token = jwt:sign(
```

Comment:
Need to call the sign with `pcall`. It is the issue of origin code, but I think we can fix it now. The error handling function can be extracted and shared between different `sign_jwt_with` functions.